src/net/sf/freecol/common/model/Unit.java

```diff
@@ -4317,6 +4317,7 @@ public int getClassIndex () {
     // Serialization
 
     private static final String ATTRITION_TAG = "attrition";
+    @SuppressWarnings("unused")
```

Comment:
Hi, curious why you're using SuppressWarnings here?

Reply:
it is currently not being used but generally sounded like on that might come in handy.

Reply:
I'm confused by that rationale, if it's not currently being used, I'd say let's drop it. We don't want to suppress warnings unless we're trying to convince the compiler that it's fine, however, this should almost never be applied at the class level, but the method where it's needed. However, again... unless explicitly needed, we shouldn't use SuppressWarnings.